app/database/queries/breeze_buddy/dispatch.py

```
@@ -0,0 +1,88 @@
+    lead_id: str, next_attempt_at: datetime
+) -> Tuple[str, List[Any]]:
+    """
+    For the manual ``/dispatch-now`` endpoint: bump ``next_attempt_at``.
+    """
+    text = f"""
+        UPDATE "{LEAD_CALL_TRACKER_TABLE}"
+        SET "next_attempt_at" = $2, "updated_at" = NOW()
+        WHERE "id" = $1
+          AND "status" = 'BACKLOG'
+        RETURNING *;
+    """
```

Ruff (0.15.12)

[error] Possible SQL injection vector through string-based query construction

(S608)